### PR TITLE
Updated serbian translation

### PR DESCRIPTION
**Pull request by CommName**

### CODE REVIEW

**Comment by flex3r** [comment]
I've added some additional ones since I also added single message deletion to the user popup. 
Can you add those as well? Looks good apart from that
```xml
<string name="user_popup_delete">Delete message</string>
<string name="confirm_user_delete_title">Confirm message deletion</string>
<string name="confirm_user_delete_message">Are you sure you want to delete this message?</string>
<string name="confirm_user_delete_positive_button">Delete</string>
```

**Comment by CommName** [comment]
Updated :)

File: app/src/main/res/values-sr/strings.xml [hunk 1]
```diff
@@ -158,4 +158,65 @@
...
+    <string name="restart">Restart</string>
+    <string name="uploader_add_header">Dodaj heder</string>
+    <string name="uploader_add_header_name">Naziv</string>
+    <string name="uploader_add_header_value">Vrednost</string>
+    <string name="uploader_title">Učitavač slika</string>
+    <string name="uploader_reset">Resetuj na početno stanje</string>
+    <string name="uploader_headers">Hederi</string>
+    <string name="uploader_image_link">Obrazac za link slike</string>
+    <string name="uploader_deletion_link">Obrazac za brisanje slike</string>
+    <string name="uploader_description">"Možete da podestie host za "</string>
```
Comment (flex3r) [inline app/src/main/res/values-sr/strings.xml:172]: Looks like this one is incomplete

File: app/src/main/res/values-sr/strings.xml [hunk 2]
```diff
@@ -158,4 +158,65 @@
     <string name="confirm_user_block_message">Da li ste sigurni da želite da blokirate korisnika?</string>
     <string name="confirm_user_block_positive_button">Blokiraj</string>
     <string name="preference_visible_badges_title">Vidljivi bedževi</string>
+    <string name="edit_dialog_title">Preimenuj kanal</string>
+    <string name="restart_required">"Potrebno je restartovati aplikaciju da biste promenili promene "</string>
```
Comment (flex3r) [inline app/src/main/res/values-sr/strings.xml:162]:
```suggestion
    <string name="restart_required">Potrebno je restartovati aplikaciju da biste promenili promene</string>
```

**Review by flex3r — CHANGES_REQUESTED** [review]

**Comment by CommName** [comment]
I made suggested changes. Sorry for taking too long, I was unavailable for the last week due to holidays

**Review by flex3r — APPROVED** [review]
All good, hope you enjoyed your holidays

**Pull request merged**